res/style.css

```diff
@@ -195,13 +194,16 @@ body, #root, .profileViewer {
 }
 
 .treeViewHighlighting {
-  padding: 1px;
-  margin: -1px;
```

Comment:
The issue was because of the right and left padding: the available space wasn't big enough anymore. Maybe there's a bug in gecko...

Reply:
see http://jsbin.com/dimoroj/edit?html,css,output

Reply:
Thanks for the comment on this.